kale/loaddata/tabular_access.py

```diff
@@ -0,0 +1,45 @@
+"""
+Read tabular data from disk. A single table can naturally represent examples
+with many features along a single dimension as rows. Examples can be labeled as
+belonging to distinct domains, subjects, and categories with additional columns.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+class UnbalancedDesignError(Exception):
+    def __init__(self, value_counts_dict: dict):
+        info_string = " ".join([f"{k}: {v}" for k, v in value_counts_dict.items()])
+        super().__init__(f"Data is not balanced across domains, subjects, and categories.\n{info_string}")
+
+
+def read_csv_tabular(csv_path: str, index_col: list) -> np.array:
+    """Read numeric data from comma separated text.
+
+    Args:
+        csv_path (str): Path to comma separated text file, with one row per example.
+        index_col (list): Columns that should be treated as levels of a Pandas
+            Dataframe MultiIndex. There should be an level for each subject,
```

Comment:
At https://github.com/pykale/pykale/blob/main/setup.cfg#L8, you can see that we use max-line-length of 120, which is more common nowadays (e.g. in PyTorch). You are probably using the default value 80(?) in black formatting. I suggest you to change the setting to 120 to be consistent in our coding style.